src/peft/tuners/lora/model.py

```diff
@@ -159,18 +160,27 @@ def _create_and_replace(
         target,
         target_name,
         parent,
-        **optionnal_kwargs,
+        current_key=None,
```

Comment:
No need to set a default of `None` if it's not an allowed value.

Reply:
addressed in latest commit